### ISSUE TITLE
Use a custom page is empty

### ISSUE THREAD

**Issue by hisham** [scullyio/scully#1399]
### Description

See https://scully.io/docs/Reference/plugins/custom-plugins/use-a-custom-plugin/. Page is empty. Is this intentional?

**Comment by SanderElias** [comment]
Nope, it is not intentional. I'm going to look at what happened to the content we did have there.
